app/assets/javascripts/tooltip.coffee

```diff
@@ -0,0 +1,12 @@
+#
+# This code uses jquery-ui to replace the built in tooltip popup with a nicer
+# one. The popup can be styled but by default is slightly larger and pops up
+# faster. The tooltip function only needs to be called once as it searches
+# the DOM attaching the new UI to any tag with a defined tooltip.
+#
+
+$(document).ready ->
+  if $("#bulk-updates-form").length < 1
```

Comment:
This means that the tooltip will only show up on pages that have elements with the `#bulk-updates-form` id. I'm not sure that is what we want...

Reply:
`< 1`, so this does work. I find it confusing too, maybe `unless $("#bulk-updates-form").length > 1`

Reply:
ah - I misread it. I like Matthew's proposed change.

Reply:
done